src/components/CurriculumComponents/DocxPOC/DocxPOC.tsx

```diff
@@ -1,16 +1,79 @@
 import { patchDocument } from "@btitterington_org/docx";
+import { Element } from "xml-js";
 
 import { download } from "./util";
 import {
   subjectPatch,
   curriculumExplainerPatch,
   linkToNewUnitTablePatch,
   subjectPrinciplesListPatch,
-  unitListTablePatch,
+  buildUnitsTable,
 } from "./patches";
 import POCForm, { POCFormData } from "./components/POCForm";
+import { STUB_UNITS_DATA } from "./patches/stubData";
+import { docToJson, findAnyReplaceByName } from "./docx";
 
 export default function DocxPOC() {
+  const onGenerateDemoFile = (
+    type: "unit_table",
+    formData: POCFormData,
+    file: File,
+  ) => {
+    if (type !== "unit_table") {
+      return;
+    }
+    const reader = new FileReader();
+
+    reader.onload = async function (event) {
+      if (!event.target?.result) {
+        return;
+      }
+      const fileContent = event.target.result as ArrayBuffer;
+      const uint8Array = new Uint8Array(fileContent);
+
+      const docPass1 = await docToJson(
+        uint8Array,
+        "w:document",
+        async (doc) => {
+          return await findAnyReplaceByName(doc, "w:body", async (el) => {
+            return {
+              ...el,
+              elements: [
+                ...(await buildUnitsTable(STUB_UNITS_DATA)),
+                ...(el.elements ?? []),
+              ],
+            } as Element;
+          });
+        },
+      );
+
+      const docPass2 = await docToJson(docPass1, "w:styles", async (doc) => {
+        return await findAnyReplaceByName(doc, "w:docDefaults", async () => {
+          return {
+            type: "element",
+            name: "w:style",
+            elements: [
+              {
+                type: "element",
+                name: "w:name",
+                attributes: {
+                  "w:val": "DEFAULT_STYLES",
+                },
+              },
+              // FIXME: We should not be nuking the default styles here...
```

Comment:
This needs fixing prior to merge

Reply:
Looks like we inherit these styles from the "default styles" of the document

<img width="534" alt="Screenshot 2024-05-07 at 11 36 01" src="https://github.com/oaknational/Oak-Web-Application/assets/235915/789419c5-c654-4ff0-9013-45d41d98cfe8">

Anybody any idea where those styles are originally defined in the word document?

Reply:
Ah I see what's going on, each paragraph is given `-1.25 cm` rather than the document margins being changed 

<img width="545" alt="Screenshot 2024-05-07 at 11 40 51" src="https://github.com/oaknational/Oak-Web-Application/assets/235915/800501fe-9acc-420d-a418-7068527e3a84">